cranelift-codegen/src/legalizer/mod.rs

```diff
@@ -65,6 +65,8 @@ fn legalize_inst(
     } else if opcode.is_branch() {
         split::simplify_branch_arguments(&mut pos.func.dfg, inst);
     } else if opcode == ir::Opcode::Isplit {
+        use crate::alloc::string::ToString;
+        dbg!(pos.func.dfg.display_inst(inst, isa).to_string());
```

Comment:
Did you want to leave this `dbg!` in? Perhaps this could be a `debug!` or `trace!` log message with some explanatory text.

Reply:
Forgot about it.